core/ui/EditorToolbar/underline.tid

```diff
@@ -11,4 +11,5 @@ shortcuts: ((underline))
 	$param="wrap-selection"
 	prefix="__"
 	suffix="__"
+	trimSeletion="yes"
```

Comment:
@pmario this typo repeats itself and makes it difficult to test.

Reply:
The typo has been everywhere, so the function was working but with the typo in the parameter. I changed it. So it should be fixed now.